### PR TITLE
Changes before o-week

### PR DESCRIPTION
**Pull request by erfanio**
also json 1.8.3 didn't compile for some reason so I just modified the lock file

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -49,50 +49,65 @@
...
-          <img class="img-circle" src="{{site.url}}/assets/img/person/sophie-huynh.jpg" />
-          <p class="name">Sophie Huynh</p>
+          <img class="img-circle" src="{{site.url}}/assets/img/person/lee-hatton.jpg" />
+          <p class="name">Lee Hatton</p>
           <p class="role">Vice-President</p>
         </div>
         <div class="col-md-3 col-xs-6">
-          <img class="img-circle" src="{{site.url}}/assets/img/person/gabriel-huynh.jpg" />
-          <p class="name">Gabriel Huynh</p>
+          <img class="img-circle" src="{{site.url}}/assets/img/person/anon.jpg" />
```
Comment (terencehuynh) [inline about.html]: _*le sigh*_

**Comment by terencehuynh** [comment]
You couldn't choose another anon image?

**Comment by erfanio** [comment]
It's just a joke one until we add the real photos, I'm not actually gonna put this up!

**Comment by terencehuynh** [comment]
But you could have done with so much better... https://www.facebook.com/OfficialNicolasCagesFaceOnThings/

**Comment by erfanio** [comment]
I'm so stealing this idea.
Maybe even an easter egg.

**Comment by erfanio** [comment]
TODO

- [x] new committee

- [ ] photos for the new committee

- [ ] archive AGM page

- [ ] upcoming events

- [ ] sponsors TBD

**Pull request merged**